api/src/main/java/org/killbill/billing/junction/BillingInternalApi.java

```diff
@@ -34,5 +33,5 @@
      *
      * @return an ordered list of billing event for the given accounts
      */
-    public BillingEventSet getBillingEventsForAccountAndUpdateAccountBCD(UUID accountId, DryRunArguments dryRunArguments, LocalDate cutoffDt, InternalCallContext context) throws CatalogApiException, AccountApiException, SubscriptionBaseApiException;
+    public BillingEventSet getBillingEventsForAccountAndUpdateAccountBCD(UUID accountId, DryRunArguments dryRunArguments, InternalCallContext context) throws CatalogApiException, AccountApiException, SubscriptionBaseApiException;
```

Comment:
I am very confused about this change. The state of [work-for-release-0.23.x](https://github.com/killbill/killbill/blob/work-for-release-0.23.x/api/src/main/java/org/killbill/billing/junction/BillingInternalApi.java#L37) shows that this parameter should be there...

Reply:
@xsalefter I suspect there was something off with the merge.